hadoop-cloud/pom.xml

```diff
@@ -201,13 +201,22 @@
   </dependencies>
 
   <profiles>
-
+    <!--
+     hadoop-3.2 profile is activated by default so hadoop-2.7 profile
+     also needs to be declared here for building with -Phadoop-2.7.
+    -->
+    <profile>
+      <id>hadoop-2.7</id>
+    </profile>
     <!--
      Hadoop 3 simplifies the classpath, and adds a new committer base class which
      enables store-specific committers.
     -->
     <profile>
       <id>hadoop-3.2</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
BTW this has no effect if _any_ other profiles are activated manually, IIRC. Does that pose a problem?

Reply:
I think it's true within each pom.
https://maven.apache.org/ref/3.3.9/maven-model/apidocs/org/apache/maven/model/Activation.html#setActiveByDefault(boolean)

In fact, tests run with `build/sbt -Phadoop-cloud -Pyarn -Phive -Phive-thriftserver -Pkubernetes "testOnly org.apache.spark.internal.io.cloud.*"`